apps/openmw/main.cpp

```diff
@@ -3,9 +3,12 @@
 #include <components/files/escape.hpp>
 #include <components/fallback/validate.hpp>
 #include <components/debug/debugging.hpp>
+#include <components/misc/rng.hpp>
 
 #include "engine.hpp"
 
+#include <chrono>
```

Comment:
This header doesn't seem to be necessary.

Reply:
Agreed, this is only really needed for rng.cpp

Reply:
Fixed